packages/popup-component/src/components.d.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable */
+/* tslint:disable */
+/**
+ * This is an autogenerated file created by the Stencil compiler.
+ * It contains typing information for all components that exist in this project.
+ */
+
+
+import { HTMLStencilElement, JSXBase } from '@stencil/core/internal';
+
+
+export namespace Components {
+  interface DcPopup {
+    /**
+    * Date
+    */
+    'date': string;
+    /**
+    * Hero
```

Comment:
Just small comment "hero" doesn't seems to be a easy name we can related to "what it represents within the popup", I would rather use "title"

Reply:
title is reserved in Stenciljs, that’s why I changed it to hero. Open to suggestions on better naming